src/main/java/org/spongepowered/common/event/SpongeEventManager.java

```diff
@@ -257,6 +257,10 @@ protected static boolean post(Event event, List<RegisteredListener<?>> handlers)
     public boolean post(Event event) {
         return post(event, getHandlerCache(event).getListeners());
     }
+    
+    public boolean post(Event event, boolean allowClientThread) {
+        return post(event);
```

Comment:
For SpongeForge to override it. See the change in `SpongeForgeEventManager` in the SpongeForge PR.